src/rt/gtfsrt_update.cc

```diff
@@ -76,8 +77,15 @@ std::ostream& operator<<(std::ostream& out, statistics const& s) {
                     s.alert_total_informed_entities_);
   print_if_no_empty("alert_invalid_route_type", s.alert_invalid_route_type_,
                     s.alert_total_informed_entities_);
-  print_if_no_empty("unsupported_vehicle", s.unsupported_vehicle_, true);
   print_if_no_empty("unsupported_no_trip_id", s.unsupported_no_trip_id_, true);
+  print_if_no_empty("no_vehicle_position", s.no_vehicle_position_, true);
+  print_if_no_empty("vehicle_position_without_position", s.vehicle_position_without_position_,
+                    true);
+  print_if_no_empty("vehicle_position_without_trip", s.vehicle_position_without_trip_, true);
+  print_if_no_empty("vehicle_position_trip_without_trip_id", s.vehicle_position_trip_without_trip_id_,
+                    true);
+  print_if_no_empty("vehicle_position_trip_without_route_id", s.vehicle_position_trip_without_route_id_,
+                    true);
```

Comment:
percentages should be relative to vehicles and only vehicles should be relative to total

Reply:
that would make sense - but could also be done in another PR